### PR TITLE
Delete the Jepsen tests

### PR DESCRIPTION
**Pull request by siyopao**
https://scalar-labs.atlassian.net/browse/DLT-2532

Also removed the alias to the Jepsen folder from `mkdocs.yml` and updated the Jepsen link to point to the `scalar-jepsen` repo.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -7,7 +7,6 @@ nav:
     - Database schema: schema.md
     - Schema loader: schema-loader.md
 - Design: design.md
-- Jepsen: jepsen.md
```
Comment (feeblefakie) [inline mkdocs.yml]: Let's create a new jepsen.md to point to the new repo.

Comment (siyopao) [reply mkdocs.yml]: Thank you. Added.

File: mkdocs.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ nav:
     - Database schema: schema.md
     - Schema loader: schema-loader.md
 - Design: design.md
+- Jepsen: https://github.com/scalar-labs/scalar-jepsen
```
Comment (feeblefakie) [inline mkdocs.yml]: Does it work ?

Comment (siyopao) [reply mkdocs.yml]: I don't know how to check without it being merged. In any case, according to https://www.mkdocs.org/user-guide/configuration/#nav it certainly should.

Comment (feeblefakie) [reply mkdocs.yml]: It should work but can you check it with `mkdocs serve` quickly ?
mkdocs can be installed through brew.

Comment (siyopao) [reply mkdocs.yml]: Works fine with mkdocs, version 1.0.4. It did not, however, with some older version of mkdocs.

Comment (feeblefakie) [reply mkdocs.yml]: Hmm, ok. not sure about what version of mkdocs readthedocs uses. We'll see...

**Comment by feeblefakie** [comment]
Merged and checked. It is working but it directs you to github so it might not a very good UX...

**Comment by siyopao** [comment]
Should it link directly to the readme?

**Comment by feeblefakie** [comment]
It shows the README so I think it's fine.

**Pull request merged**